app/webpacker/components/wca/EventSelector.jsx

```diff
@@ -22,26 +22,29 @@ export function EventSelector({
   // Listing event as an argument here to indicate to developers that it's needed
   // eslint-disable-next-line no-unused-vars
   disabledText = (event) => {},
+  showLabels = false,
```

Comment:
Defaulting this to false means all existing uses no longer show the title and buttons.